src/Mondu/Mondu/Presenters/PaymentInfo.php

```diff
@@ -41,6 +41,12 @@ public function get_invoices_data() {
     return $this->invoices_data;
   }
 
+  public function get_wcpdf_shop_name() {
+    $wcpdf = \WPO_WCPDF::instance();
+
+    return $wcpdf->documents->documents['\WPO\WC\PDF_Invoices\Documents\Invoice']->get_shop_name() ?? get_bloginfo('name');
```

Comment:
**question:** Will it break if the merchant does not have the `WPO_WCPDF` in their WordPress?

Reply:
If the function is called when merchant doesn't have WPO_WCPDF it will break, but we only use this one to extend the WPO_WCPDF invoice template so I believe we shouldn't have any issues.